packages/frontend/src/features/semanticViewer/components/FiltersModal/index.tsx

```diff
@@ -0,0 +1,183 @@
+import {
+    FieldType as FieldKind,
+    SemanticLayerFieldType,
+} from '@lightdash/common';
+import {
+    ActionIcon,
+    Button,
+    Flex,
+    Group,
+    Modal,
+    Select,
+    Stack,
+    type ModalProps,
+} from '@mantine/core';
+import { IconPlus, IconX } from '@tabler/icons-react';
+import { useCallback, useMemo, useState, type FC } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import MantineIcon from '../../../../components/common/MantineIcon';
+import useToaster from '../../../../hooks/toaster/useToaster';
+import { useAppDispatch, useAppSelector } from '../../store/hooks';
+import { selectAllSelectedFieldNames } from '../../store/selectors';
+import {
+    addFilter,
+    removeFilter,
+    updateFilter,
+} from '../../store/semanticViewerSlice';
+import Filter from './Filter';
+import FilterButton from './FilterButton';
+import FilterFieldSelectItem from './FilterFieldSelectItem';
+
+type FiltersModalProps = ModalProps & {
+    onApply?: () => void;
+};
+
+const FiltersModal: FC<FiltersModalProps> = ({
+    onApply,
+    onClose,
+    ...props
+}) => {
+    const [isAddingFilter, setIsAddingFilter] = useState(false);
+    const { filters, fields } = useAppSelector((state) => state.semanticViewer);
+    const allSelectedFieldNames = useAppSelector(selectAllSelectedFieldNames);
+
+    const dispatch = useAppDispatch();
+
+    const { showToastError } = useToaster();
+
+    const availableFieldOptions = useMemo(() => {
+        if (!fields) return [];
+
+        return fields
+            .filter(
+                (f) =>
+                    f.visible &&
+                    f.type === SemanticLayerFieldType.STRING &&
+                    f.kind === FieldKind.DIMENSION, // TODO: for now only string dimensions are supported
+            )
+            .map((f) => ({
+                value: f.name,
+                field: f,
+                label: f.label,
+                group: allSelectedFieldNames.includes(f.name)
+                    ? 'Results'
+                    : 'Other fields',
+            }))
+            .sort((a, b) =>
+                a.group === 'Results' && b.group !== 'Results' ? 0 : 1,
+            );
+    }, [allSelectedFieldNames, fields]);
+
+    const handleApply = useCallback(() => {
+        onApply?.();
+        onClose?.();
+    }, [onApply, onClose]);
+
+    return (
+        <Modal
+            {...props}
+            onClose={onClose}
+            overlayProps={{ opacity: 0.1 }}
```

Comment:
hmm, isn't it too low?

Reply:
Might be, it looked like setting it higher would make it look too dark 🤔 

this is how it currently looks:
![image](https://github.com/user-attachments/assets/57411e7d-81df-45fc-a688-53857596aa0c)

this is with 0.5

![image](https://github.com/user-attachments/assets/bea574bc-18c5-42e7-8f93-2564f9a20f20)

Reply:
perhaps set it to somewhere in the middle? like 0.3?